cli/formatters/pot.js

```diff
@@ -104,11 +104,23 @@ module.exports = function( matches, options ) {
 			matchId = uniqueMatchId( match );
 
 		if ( uniqueMatchesMap[ matchId ] ) {
+			if ( match.line ) {
+				if ( uniqueMatchesMap[ matchId ].line === undefined ) {
+					uniqueMatchesMap[ matchId ].line = match.line;
+				} else {
+					uniqueMatchesMap[ matchId ].line += '\n' + match.line;
```

Comment:
Let's just append to an array and join all the lines once instead of concatenating, splitting and joining.